### PR TITLE
Add the old version of insert-generator

### PR DESCRIPTION
**Pull request by OlegIvsv**

### CODE REVIEW

File: InsertGenerator/.gitignore
Comment (BloodyOcean) [inline InsertGenerator/.gitignore:1]: Please, delete this file, because we already have it

File: InsertGenerator/generators/null_generator.py
Comment (BloodyOcean) [inline InsertGenerator/generators/null_generator.py:1]: Please, add comments which explain why do we need these classes

File: InsertGenerator/generators/special_generator.py
Comment (BloodyOcean) [inline InsertGenerator/generators/special_generator.py:1]: Please, add comment how to use this class

File: InsertGenerator/main.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+def fill_ps(file_path):
+    test_data = [
+        data_for_seller(),
+        data_for_customer(),
+        data_for_order()
+    ]
+    res = ''
+    for td in test_data:
+        res += td + '\n\n'
```
Comment (BloodyOcean) [inline InsertGenerator/main.py:61]: Please, use join method. https://www.w3schools.com/python/ref_string_join.asp

File: InsertGenerator/main.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    with open(file_path, 'w') as file:
+        file.write(res)
+
+
+fill_ps(r'C:\Users\Oleh\Desktop\Studying\DB\LAB_2\SQL_sell-purchase_fill.sql')
```
Comment (BloodyOcean) [inline InsertGenerator/main.py:67]: Please, don't use hardcoded paths and variables. Use config file instead of that

File: InsertGenerator/variant3.py
Comment (BloodyOcean) [inline InsertGenerator/variant3.py:1]: Please, add docstrings for each function in order to use it by anyone else in the future

File: InsertGenerator/generators/special_generator.py
Comment (BloodyOcean) [inline InsertGenerator/generators/special_generator.py:1]: Don't forget to initialize each attribute by None in class declaration

File: InsertGenerator/generators/email_generator.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+class EmailGenerator(Generator):
+    def __init__(self, *, domains=None):
+        if domains is None:
+            domains = ['lpnu.ua', 'gmail.com', 'ukr.net']
+        self.lorem = TextLorem(wsep='.', srange=[2, 3])
+        self.domains = domains
+
+    def next(self) -> str:
+        return '\'' + self.lorem.sentence() + '@' + random.choice(self.domains) + '\''
```
Comment (BloodyOcean) [inline InsertGenerator/generators/email_generator.py:15]: Please, use string interpolation for clear

**Review by BloodyOcean — CHANGES_REQUESTED** [review]
Left some comments, please pay attention on them

**Comment by OlegIvsv** [comment]
зображення

**Review by danylohlynsky — APPROVED** [review]

**Pull request merged**